Automattic/newspack-plugin
# feat(google-oauth): display notice if there's no refresh token

### All Submissions:






### Changes proposed in this Pull Request:



~Google OAuth will not issue a refresh token if the OAuth app has already been authorised with another site.~ This is not true (see [comment](https://github.com/Automattic/newspack-plugin/pull/1217#issuecomment-963943790)). But still there might be cases when the refresh token is missing. 

### How to test the changes in this Pull Request:

1. Configure a site to use Google OAuth (`NEWSPACK_GOOGLE_OAUTH_PROXY` environment variable)
2. [Revoke](https://myaccount.google.com/permissions) the previous authentication, if present
3. Complete the OAuth flow in the Connections wizard, observe no error notice is displayed
4. ~Complete the OAuth flow on _another site_~  _Disable [this line](https://github.com/Automattic/newspack-plugin/blob/f05cc22d6f59f892ec4705be855a72accca31e1d/includes/oauth/class-google-oauth.php#L411) to mimick a refresh-token-less state_, observe an error notice prompting to revoke authentication displayed in the Connections wizard

### Other information:
